src/client/graphics/layers/FactoryRadiusLayer.ts

```
@@ -0,0 +1,125 @@
+    hasChanges ||= !isNew && info.ownerId !== ownerId;
+
+    return hasChanges;
+  }
+
+  private collectFactoryRanges(): void {
+    const activeFactories = this.game
+      .units(UnitType.Factory)
+      .filter((unit) => unit.isActive());
+
+    this.factories.clear();
+    activeFactories.forEach((factory) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)